packages/cache-handler/src/instrumentation/register-initial-cache.ts

```diff
@@ -188,6 +188,7 @@ export async function registerInitialCache(CacheHandler: CacheHandlerType, optio
             await cacheHandler.set(
                 cachePath,
                 {
+                    // @ts-expect-error
                     kind: 'ROUTE',
```

Comment:
I believe this needs to be `APP_ROUTE` now, it seems `ROUTE` has been deprecated. Metadata routes like `favicon.ico` or `manifest` were failing due to this.